ihontaryk/homework_10/context_manager_timer.py

```diff
@@ -0,0 +1,22 @@
+import time
+
+
+class Timer:
+    def __init__(self):
+        self.total = 0
+        self.start = None
+
+    def __enter__(self):
+        self.start = time.time()
+        return self
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        if self.start:
+            self.total += (time.time() - self.start)
```

Comment:
we assume that user uses with block
in such case self.start will always be there